cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -145,7 +150,6 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     private DeploymentTemplate templateDeployment;
     private PodTemplate templatePod;
     private InternalServiceTemplate templateService;
-
```

Comment:
I would keep the empty line here. It separates different types of variables here.

Reply:
Added line back